Automattic/wp-calypso
# [e2e] Updates for more stable runs

#### Changes proposed in this Pull Request

After couple of random failures last week ( [#](https://circleci.com/gh/Automattic/wp-calypso/488601) [#](https://circleci.com/gh/Automattic/wp-calypso/490638) [#](https://circleci.com/gh/Automattic/wp-calypso/491068) [#](https://circleci.com/gh/Automattic/wp-calypso/491117) ) I updated some things to make tests more stable.  

#### Testing instructions

Make sure that CircleCI tests are passing. You can run them a couple of times and see is there any inconsistency.
